### DISCUSSION TITLE
Lab 4.02 task 1

### DISCUSSION THREAD

**Discussion by chrtamayo** [rramosp/ai4eng.v1#70] [04 - GESTIÓN DE DATOS]
![imagen](https://user-images.githubusercontent.com/94641640/142697469-8d982b4f-596d-42b7-9349-52bab16ab7c6.png)

Me genera este error para la task2 y la task 3

Task 1 ProblemaSolucionado

Hola profesor.

Parte 1: Edit Solucionado (el problema era que la fecha que busque estaba erronea porque excel solo permite tener 4 hojas de 1048576 longitud y el archivo csv tiene una longitud 20216100)

Respecto a este laboratorio ejercicio 1 no me queda del todo claro como se procede en la filtracion de datos, dado a que decidi descargar las bases de datos en excel para entender mejor las bases de datos y porque en el colab se demoraba dado a que el csv train pesa 600mb, pero realizando los pasos de filtro en el excel, obtengo estos datos sobre el caso ejemplo de  building number 900, on 2016-02-01

![imagen](https://user-images.githubusercontent.com/94641640/142676726-94f94e79-0b95-4af5-9ae6-9cb481c83172.png)

y en el caso del ejemplo dice que la suma de los objetos target es de 4941 y en el filtrado me da 3223 

Parte 2:  Solucionado

en el de la suma del resto de variables dice 2705422 pero yo no se cuales son precisamente los que se deben sumar, yo sumo todos los datos en rojo y me da 145084, entonces no se como es el caso concreto, asi que espero a la respuesta antes de proceder con el python,

tambien tengo una pequeña duda armando la tabla de las 11 columnas, en el caso del square_feet y year_built en este caso solo tiene 1 row y en el caso del  year_built es nulo entonces en year built se llena todo NaN y en el de square_feet se llena de todas las rows del valor o se llena solo 1 y el resto NaN

**Comment by Felipe-RA** [comment]
Hola Chrtamayo

Veo que lograste solucionar tus dudas por tu cuenta, ¡¡me alegra!!

Al trabajar datasets hay que tener cuidado con programas como el excel, a veces pueden cambiarle la estructura a los datos dependiendo de los separadores de datos que maneje [algunos usan ( **;** ) , otros usan ( **,** ) ]

**Reply by chrtamayo** [reply]
Hola gracias por responder, ahora el error que me esta saliendo del 4.02 hasta el 5.02 es el de : grader for task does not exist

**Reply by Felipe-RA** [reply]
Saludos Chrtamayo

Gracias por el aviso, vamos a revisar la plataforma y te comentamos en cuanto tengamos una solución

**Reply by rramosp** [reply]
gracias cristian ... fue un fallo en la configuración inicial del curso. ya está solucionado. haznos saber si tienes más inconvenientes
